views/partials/modals/tasksModal.handlebars

```diff
@@ -69,19 +69,12 @@
 
               </div>
             </div>
-            {{! dateDueOn }}
-            <div class="form-floating mb-3">
-              <input type="datetime-local" class="form-control" name="dateDueOn" id="task_dateDueOn" value="" />
-              <label for="task_dateDueOn" class="form-label">Date event is
-                due on
-              </label>
-
-              <div id="task_dateDueOn_error" class="invalid-feedback">
-
-              </div>
-            </div>
-
             {{! add more fields for other events }}
+            <div class="mb-3">
+              <label for="task_checked" class="form-label">Completed</label>
+              <input type="checkbox" class="form-check-input" name="checked" id="task_checked" value="true" />
+              <div id="task_checked_error" class="invalid-feedback">
+              </div>
 
           </form>
```

Comment:
check html page with w3 schools validator once to make sure we dont have any duplicate ids or incorrect html
https://validator.w3.org/nu/#textarea

Reply:
valid source now